packages/renderer/src/lib/engine/Prune.svelte

```diff
@@ -27,7 +27,7 @@ async function prune(type: string) {
     case 'containers':
       engines.forEach(async engine => {
         try {
-          await window.pruneContainers(engine.id);
+          window.pruneContainers(engine.id);
         } catch (error) {
           console.error(error);
```

Comment:
Should we add the engine.id here to be sure we can associate the error message to it?

Reply:
Error includes engine.id I believe, but I can double check.